dockernet/config/ica_host.json

```diff
@@ -17,6 +17,7 @@
                     "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation",
                     "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
                     "/cosmos.distribution.v1beta1.MsgSetWithdrawAddress",
```

Comment:
Is MsgFundCommunityPool enabled in the "add ICA to DYDX" PR?

Reply:
There aren’t any messages specified in that PR cause it will go in the upgrade handler, but Aidan did send a list of messages to allow that included this one